hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/bootstrap/SparkBootstrapCommitActionExecutor.java

```diff
@@ -222,6 +225,17 @@ protected void commit(Option<Map<String, String>> extraMetadata, HoodieWriteMeta
     LOG.info("Committing metadata bootstrap !!");
   }
 
+  @Override
+  protected void syncTableMetadata() {
```

Comment:
I think all this can be removed. from all action executors? can't we take the lock in post commit/write from the write client level?

Reply:
can we please file a ticket for removing this auto commit stuff. it's kind of messy.

Reply:
https://issues.apache.org/jira/browse/HUDI-1665